tasks/variables/variables.js

```diff
@@ -1,40 +1,49 @@
 /**
  * TODO create variable named int and set value equal to 1
 */
-
+let int = 1;
```

Comment:
If the value is not supposed to be changed, please use 'const', otherwise, 'let'